plugin-server/src/types.ts

```diff
@@ -140,6 +140,8 @@ export interface PluginsServerConfig extends Record<string, any> {
 
 export interface Hub extends PluginsServerConfig {
     instanceId: UUID
+    // Capabilities for this hub
```

Comment:
nit: this comment is kind of useless - a capabilities field inside Hub tells me these are the capabilities for the Hub already.

I'd be more interested as a reader in what "capabilities" means

Reply:
Improved it a bit. Note I'm following convention from rest of this type.